CHANGELOG.md

```diff
@@ -46,7 +46,7 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
 ### Removed
 - `Text.aspect_ratio`, it now matches the viewport aspect ratio
 - `GraphicConfig.scissor_test` parameter
-- Support for Android < 9.0
+- `ngl-viewer` and `ngl-diff` support and release on Apple platforms
```

Comment:
The diff is broken

Also, I think ngl-diff and ngl-viewer are very useful so I'd rather not drop them, unless you have a solution coming up soon.

Reply:
> The diff is broken

Fixed locally.

> 
> Also, I think ngl-diff and ngl-viewer are very useful so I'd rather not drop them, unless you have a solution coming up soon.

Keeping compatibility with old GL versions is a maintenance burden I don't want to carry anymore, especially on a platform where GL has been deprecated for long time now. Dropping support of old GL version will speed up the development of the NGPU code. Internal users (you, me) won't have to worry about restricting themselves from using compute, storage buffers, images, etc where relevant.

While a solution is found for these tools on macOS. One can still use the current release of such tools for the time being.